test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/main.py

```diff
@@ -362,6 +364,7 @@ def _check_interpreter(self, powershell=False):
         if not self.text.startswith('#!/usr/bin/python'):
             self.reporter.error(
                 path=self.object_path,
+                name='interpreter-line-python',
```

Comment:
I think we can remove this check in a separate PR, as the shebang sanity test covers this.